homeassistant/components/elkm1/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "elkm1",
   "name": "Elk-M1 Control",
   "documentation": "https://www.home-assistant.io/integrations/elkm1",
-  "requirements": ["elkm1-lib==0.8.8"],
+  "requirements": ["elkm1-lib==0.8.10"],
```

Comment:
Please rebase when the other PR is merged.

Reply:
Is the PR waiting for a rebase? If so, I don't know how to rebase.

Reply:
Here are instructions.

https://developers.home-assistant.io/docs/development_catching_up

If that doesn't work it's ok to start over in a new PR too. Thanks!